pom.xml

```diff
@@ -1,14 +1,59 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
   <groupId>io.dapr</groupId>
-  <artifactId>client</artifactId>
-  <packaging>jar</packaging>
-  <version>0.1.0-preview01</version>
-  <name>dapr-client</name>
+  <artifactId>client-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>0.1.0-preview02</version>
+  <name>dapr-client-parent</name>
   <description>gRPC client for Dapr.</description>
   <url>https://dapr.io</url>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <grpc.version>1.25.0</grpc.version>
+    <protobuf.version>3.11.0</protobuf.version>
+    <protoc.version>3.10.0</protoc.version>
+    <os-maven-plugin.version>1.6.2</os-maven-plugin.version>
+    <maven-dependency-plugin.version>3.1.1</maven-dependency-plugin.version>
+    <maven-antrun-plugin.version>1.8</maven-antrun-plugin.version>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
I understand that Java 11 is the latest. On the other hand, this is a SDK to be consumded by existing applications as well. If we move to Java 11, would it break compatibility with previous version? If so, then we should move down to Java 10, 9 or maybe even 8 to make sure we have a decent coverage for existing apps. I searched and found one report from 2018 and this one for 2019: https://www.jetbrains.com/lp/devecosystem-2019/java/

It seems that Java 8 is still the most popular SE by a wide margin.